entrypoint.sh

```diff
@@ -1,10 +1,4 @@
 #!/bin/sh
```

Comment:
Yep, this pr is breaking change, because we split bundle into app chunk and vendor chunk, user will have to modify setup script to load vendor anyway, or it won't work.

If you don't like breaking change, we can leave bundle splitting for 2.0 release and only make respect `host` property, which will be backwards compatible.

Reply:
agree. we don't want to introduce a breaking change yet. Let's try to fix the actual `host` issue for now and postpone the revolution ;)

Reply:
Ok, got it, so, I'm gonna put down this pr. I'm going to make 2 new ones. One for `host`, and second one is about `delete me` feature, be glad If you help me with this one, see first comment.